doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md

```diff
@@ -33,14 +34,17 @@ Deprecated features are removed not before 3 development sprints (usually, a dev
 [Top](#top)
 
 ## <a name="section2"></a>Deprecated functionalities
-### <a nane="section2.1"></a>Grouping Rules
+### <a nane="section2.1"></a>NGSIv1 support
+Deprecated after releasing version [1.18.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/1.18.0).
```

Comment:
The link is not still working but it will when 1.18.0 will finally released

NTC (informative)